File: Selenium/Activities/Activity1.py
from selenium import webdriver
from selenium.webdriver.common.by import By

# Create driver
driver = webdriver.Firefox()

# Open website
driver.get("https://training-support.net")

# Print current page title
print("Page title is:", driver.title)

# Click "About Us" link
driver.find_element(By.LINK_TEXT, "About Us").click()

# Print new page title
print("New page title is:", driver.title)

# Close browser
driver.quit()

File: Selenium/Activities/Activity1.java
package examples;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity1 {
    public static void main(String[] args) {

        // Step 1: Launch browser
        WebDriver driver = new FirefoxDriver();

        // Step 2: Open website
        driver.get("https://training-support.net");

        // Step 3: Print page title
        System.out.println("Page title is: " + driver.getTitle());

        // Step 4: Click "About Us" link
        driver.findElement(By.linkText("About Us")).click();

        // Step 5: Print new page title
        System.out.println("New page title is: " + driver.getTitle());

        // Step 6: Close browser
        driver.quit();
    }
}